js/admin/notifications.js

```diff
@@ -116,7 +116,7 @@ function getPush()
 				// Add customers notifications to the list
 				html = "";
 				$.each(json.customer.results, function(property, value) {
-					html += "<a class='notif' href='" + value.customer_view_url + "'>";
+					html += "<a class='notif' href='"+baseAdminDir+"index.php/sell/customers/" + parseInt(value.id_customer, 10) +"/view?_token=" + token_admin_customers + "'>";
```

Comment:
it's not a good idea to have route in plain text in the javascript. Please use the router component instead